scm-webapp/src/main/java/sonia/scm/repository/RepositoryInitializer.java

```diff
@@ -55,10 +57,14 @@ public RepositoryInitializer(RepositoryServiceFactory serviceFactory, Set<Reposi
   }
 
   public void initialize(Repository repository) {
+    initialize(repository, null);
```

Comment:
Is `null` a good option here? Do we need this method? Could we not just remove the whole method?

Reply:
I thought it would be a breaking change to remove this method.